### ISSUE TITLE
Bug: cannot omit `resolve` in subscription resolver

### ISSUE THREAD

**Issue by zhaoyao91** [apollographql/apollo-server#2741]
- version: apollo-server@2.5.0
- demo repo: https://github.com/zhaoyao91/apollo-issue-demo-001

please see [this line](https://github.com/zhaoyao91/apollo-issue-demo-001/blob/master/src/index.js#L44). Thought the [doc](https://www.apollographql.com/docs/graphql-subscriptions/subscriptions-to-schema/#payload-transformation) I thought this is optional if the payload doesn't need any transformation. But if I remove this line, the subscription always responds `null` data.

**Comment by KeithGillette** [comment]
I just spent hours coming to the same conclusion trying to get Subscriptions working with `apollo-server-express@2.19.0`!

The [Apollo Server Subscriptions documentation](https://www.apollographql.com/docs/apollo-server/data/subscriptions/) doesn't even mention the `resolver` property. It wasn't until I dug into the [`graphql-subscriptions` package documentation](https://github.com/apollographql/graphql-subscriptions) and added pass-through `resolver` just to see what was happening that I stopped getting the `Error: Cannot return null for non-nullable field Subscription.<SubscriptionName>` error.
